pkg/service/manifest/response.go

```diff
@@ -0,0 +1,100 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/credential/manifest"
+
+	cred "github.com/tbd54566975/ssi-service/internal/credential"
+	"github.com/tbd54566975/ssi-service/internal/keyaccess"
+	"github.com/tbd54566975/ssi-service/internal/util"
+	"github.com/tbd54566975/ssi-service/pkg/service/credential"
+	"github.com/tbd54566975/ssi-service/pkg/service/keystore"
+)
+
+func (s Service) signResponseJWT(signingDID string, r CredentialResponseContainer) (*keyaccess.JWT, error) {
```

Comment:
maybe rename signCredentialResponseJWT for clarity

Reply:
done